src/JWT.php

```diff
@@ -132,8 +133,8 @@ public static function decode(
             // See issue #351
             throw new UnexpectedValueException('Incorrect key for this algorithm');
         }
-        if ($header->alg === 'ES256' || $header->alg === 'ES384') {
-            // OpenSSL expects an ASN.1 DER sequence for ES256/ES384 signatures
+        if (in_array($header->alg, ['ES256', 'ES256K', 'ES384'])) {
```

Comment:
Fix style check and also make type strict
```suggestion
        if (\in_array($header->alg, ['ES256', 'ES256K', 'ES384'], true)) {
```